src/components/Logo/logo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Logo = (props) =>{
+    return(
+        <img width="50" height="50" src="../../assets/images/logo-320.svg" alt="Fetch Logo"/>
```

Comment:
Size of ur logo is always gonna be same as that.?  Make it modular and extensible.

Reply:
We can implement that thing later on right now i dont thing we will be required to change logo sizes